app/Http/Controllers/V2/Terrafund/TerrafundClipGeometryController.php

```diff
@@ -23,10 +24,11 @@ public function clipOverlappingPolygonsBySite(string $uuid)
         ini_set('memory_limit', '-1');
         $user = Auth::user();
         $polygonUuids = GeometryHelper::getSitePolygonsUuids($uuid)->toArray();
-        $job = new FixPolygonOverlapJob($polygonUuids, $user->id);
+        $delayedJob = DelayedJob::create(['status' => DelayedJob::STATUS_PENDING]);
```

Comment:
It's a little weird to me that setting `status` is required. Can we make it a default value on the DB column definition instead? I don't think we'd ever have a use case where we create a delayed job in another status.

Reply:
Made pending as default. changed the creation for all ocurrences